server/src/test_util.rs

```diff
@@ -0,0 +1,82 @@
+use crate::db_connection::DBClient;
```

Comment:
Just put a `![cfg(test)]`  on this entire module?

Reply:
fixed in latest commit